apps/labs/posts/numpy-quaddtype-blog.md

```diff
@@ -0,0 +1,419 @@
+</figure>
+
+1. <u>Absolute Error in Energy Level Differences</u>
+
+    - **Quad Precision**: Maintains extremely low error, consistently around 10<sup>-52</sup> to 10<sup>-53</sup> J across all quantum numbers, demonstrating exceptional stability and accuracy.
+    - **Float64**: Shows significantly higher error, ranging from 10<sup>-34</sup> to 10<sup>-33</sup> J, with a slight upward trend for higher quantum numbers, indicating gradual precision loss.
+
+2. <u>Absolute Error in Wavefunction Normalization</u>
+
+    - **Quad Precision**: Exhibits remarkably stable error around 3.65 × 10<sup>-13</sup>, ensuring consistent wavefunction validity across all quantum states.
+    - **Float64**: Starts near quad precision levels but shows a clear upward trend for higher states, suggesting accumulating inaccuracies.
+
+## Current Status And Next Steps
+
+As of this writing, `numpy_quaddtype` has reached a significant milestone. It's ready for use, supporting most major operations and integrating well with NumPy. As we've demonstrated with our Mandelbrot set visualizations and quantum harmonic oscillator calculations, it's capable of handling demanding computational tasks that require high precision.
+
+However, like any new software project, there's still room for improvement and expansion. During development, we identified several areas in NumPy's dtype C-API that could be enhanced to better support custom dtypes like ours ([NumPy Issue #27231](https://github.com/numpy/numpy/issues/27231))
+
+Looking ahead, we have several key objectives:
+
+1. **Package Distribution**: We're preparing to release `numpy_quaddtype` as Python wheels, which will be available for installation via **PyPI** and as a **conda package**. This will make it easily accessible to the wider Python scientific computing community.
+
+2. **Community Engagement**: We plan to make a public announcement about `numpy_quaddtype` and actively seek feedback from the community. User experiences and suggestions will be crucial for guiding future improvements.
+
+3. **NumPy Enhancement Proposal (NEP)**: We're drafting a NEP to formally propose the integration of `numpy_quaddtype` into the NumPy ecosystem. This process will allow for thorough discussion and review by the NumPy community.
+
+4. **Optimized Linear Algebra**: One of our next technical challenges is to support optimized linear algebra routines for our quadruple precision dtype. This will enhance performance for matrix operations, which are crucial in many scientific and engineering applications.
+
+5. **Long-term NumPy Integration**: As `numpy_quaddtype` matures and gains wider adoption, we aim to work with the NumPy community to explore the possibility of it eventually replacing `np.longdouble`. This would provide a more consistent, cross-platform solution for high-precision calculations.
```

Comment:
I'd delete bullet point #5, IMO this is covered by #3. You'd be writing a NEP to eventually replace `np.int128` with a `tlfloat`-backed quad dtype on all platforms.